api/src/controllers/team.js

```diff
@@ -292,11 +292,17 @@ async function deployToGHPages(req, res, next) {
 async function updateTeam(req, res, next) { // eslint-disable-line no-unused-vars
   const { team_id: _id } = req.params;
   const team = req.body;
+  if (team.password) {
+    const salt = await bcrypt.genSalt();
+    const hashedPassword = await bcrypt.hash(team.password, salt);
+    team.password = hashedPassword;
+  }
 
   const updatedTeam = await Team.findByIdAndUpdate(_id, team, {
     new: true,
     runValidators: true,
   });
+
   res.status(200).json(updatedTeam);
```

Comment:
The hashed password is returned here, not sure if that's safe. Also, if the request to this endpoint has the password in plain text so if someone happens to intercept the pwd is exposed, is this an issue? @pmyip3 @aksh0001

Reply:
I think @pmyip3 left a comment on that but it’s been marked resolved, so I believe we’re deleting it before sending back